easyrest/models/__init__.py

```diff
@@ -17,6 +17,7 @@
 from .tag import Tag
 from .tag_association import TagAssociation
 from .user import User
+
```

Comment:
What is the purpose of this change?

Reply:
It`s about space? If yes - I accidentally added this.